script/example.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+import yaml
+import json
+from argparse import ArgumentParser
+
+def action_json(args):
+
+    with open('scripts/example_data.json') as json_file:
```

Comment:
Consider using 'with open' in combination with 'json.load' directly instead of 'json.loads' with 'read' to simplify the code and handle file reading more efficiently.
```suggestion
        print(json.dumps(json.load(json_file), indent=2))
```